examples/federated-database-query-limit/variables.tf

```
@@ -0,0 +1,90 @@
+variable "public_key" {
+  type        = string
+  description = "Public Programmatic API key to authenticate to Atlas"
+}
+variable "private_key" {
+  type        = string
+  description = "Private Programmatic API key to authenticate to Atlas"
+}
+variable "project_id" {
+  type        = string
+  description = "MongoDB Project ID"
+}
+
+variable "collection_1" {
+  type        = string
+  description = "Human-readable label that identifies the collection in the database in first cluster"
+}
+
+variable "database_1" {
+  type        = string
+  description = "Human-readable label that identifies the database, which contains the collection in the first cluster"
+}
+
+variable "collection_2" {
+  type        = string
+  description = "Human-readable label that identifies the collection in the database in second cluster"
+}
+
+variable "database_2" {
+  type        = string
+  description = "Human-readable label that identifies the database, which contains the collection in the second cluster"
+}
+
+variable "federated_instance_name" {
+  type        = string
+  description = "MongoDB Federated Database Instance Name."
+  default     = "FederatedDatabaseInstance0"
+}
+
+variable "atlas_cluster_name_1" {
+  type        = string
+  description = "First Atlas Cluster Name."
+  default     = "ClusterFederatedTest1"
+}
+
+variable "atlas_cluster_name_2" {
+  type        = string
+  description = "First Atlas Cluster Name."
```

should this be "Second"? 
```suggestion
  description = "Second Atlas Cluster Name."
```